.github/workflows/container-autoupdate.yml

```diff
@@ -39,12 +39,6 @@ jobs:
             tests/pylint/runpylint*.log
             tests/coverage-*.log
 
-      - name: Fail if local test did not worked
-        if: ${{ steps.run-tests.outcome == 'failure' }}
-        run: |
-          cat tests/test-suite.log
```

Comment:
Don't we want to have this `cat` call as part of ``run-tests`` step?

Reply:
Good point, thanks! Fixed.